rust/noosphere-common/src/channel.rs

```diff
@@ -31,14 +39,17 @@ impl From<RecvError> for ChannelError {
     }
 }
 
-/// Represents a request to be processed in `MessageProcessor`,
-/// sent from the associated `MessageClient`.
+/// Represents a request to be processed in [MessageProcessor],
+/// sent from the associated [MessageClient].
```

Comment:
A doc example might be a nice stand-in for tests, just to demonstrate intended usage.

Reply:
Ah, nevermind, I see there is a test!